docs/mutect/mutect.tex

```diff
@@ -219,23 +220,35 @@ \section{Strand Artifact Model}
 
 
 \section{Germline Filter}\label{germline-filter}
-Suppose we have detected an allele such that its (somatic) likelihood in the tumor is $\ell_t$ and its (diploid) likelihood in the normal is $\ell_n$.  By convention, both of these are relative to a likelihood of $1$ for the allele \textit{not} to be found.  If we have no matched normal, $\ell_n = 1$.  Suppose we also have the population allele frequency $f$ of this allele.  Then the prior probability for the normal to be heterozygous or homozygous alt for the allele is $2f(1-f) + f^2$ and the prior probability for the normal genotype not to contain the allele is $(1-f)^2$.  Finally, suppose that the prior for this allele to arise as a somatic variant is $\pi$.
+Suppose we have detected an allele such that its (somatic) likelihood in the tumor is $\ell_t$ and its (diploid) likelihood in the normal is $\ell_n$.  By convention, both of these are relative to a likelihood of $1$ for the allele \textit{not} to be found.  If we have no matched normal, $\ell_n = 1$.  Suppose we also have the population allele frequency $f$ of this allele.  Then the prior probabilities for the normal to be heterozygous and homozygous alt for the allele are $2f(1-f)$ and $f^2$ and the prior probability for the normal genotype not to contain the allele is $(1-f)^2$.  Finally, suppose that the prior for this allele to arise as a somatic variant is $\pi$.
 
 We can determine the posterior probability that the variant exists in the normal genotype by calculating the unnormalized probabilities of four possibilities:
 \begin{enumerate}
-\item The variant exists is both the normal and the tumor samples.  This has unnormalized probability $\left(2f(1-f) + f^2 \right) \ell_n \ell_t (1 - \pi)$.
+\item The variant exists in the tumor and the normal as a germline het.  This has unnormalized probability $2f(1-f) \ell_n \ell_t (1 - \pi)$.
+\item The variant exists in the tumor and the normal as a germline hom alt.  This has unnormalized probability $f^2 \ell_n \ell_t (1 - \pi)$.
```

Comment:
`ell_n` is the total likelihood to be present, as either het or hom alt, in the normal.  I added a footnote that clarifies this.

One could argue that we use some power by not modeling the normal more precisely i.e. the model as it stands allows a variant to be a germline het in the normal and a germline hom alt in the tumor.  However, when we have a matched normal and `ell_n` isn't small we always filter the variant regardless of anything else and the details of the model don't matter.  They only matter when we're in tumor-only mode (or have no coverage in the normal), in which case `ell_n` is basically deactivated

Reply:
Got it, the variant won't even be emitted when normal is hom alt. 

But in the code, `\ell_n` seems to be coming from `SomaticGenotypingEngine::diploidAltLog10Odds`, which assumes that the germline variant is het, so wouldn't it be incorrect to say that `\ell_n` is the total likelihood for het or hom alt?

Reply:
I see your point.  That method is the log odds of het vs hom ref, but if the variant is hom alt these log odds will also be overwhelmingly large, because het is so much closer to the truth than hom ref.  So really, in a slightly sloppy way, `\ell_n` really *is* coming from both het and hom alt.

As a germline genotyper it's too primitive, but for Mutect's purposes it's good enough.